src/main/java/objectsBasicsMandatoryPracticalTasks/Dogs/Dog.java

```diff
@@ -0,0 +1,44 @@
+package objectsBasicsMandatoryPracticalTasks.Dogs;
+
+class Dog {
+    private String name, age, size;
+
+    Dog(String name, String age, String size) {
```

Comment:
Would be better to store `age` as int and `size` as enum, so that no need to convert it each time when you want to get these values in `getAge` or `getSize`

Reply:
The conversion might be applied externally in `DogsDemo` for example